RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
`VPSet::copy` just does a shallow copy
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/FWCore/ParameterSet/python/Types.py#L940

Since this code is changing the items in the container (rather than changing what items are in the container) I suggest switching to `copy.deepcopy` to recursively copy the container and its contents.